cypress/integration/Alert.feature

```diff
@@ -0,0 +1,131 @@
+Feature: Alert component
```

Comment:
The length of this file might be a readability issue, is there a way to split it into separate files?

Reply:
I don't think it makes sense to split the file when all the scenarios relate to the same component. Is 131 lines excessive? We've ran this file several times without issue.

Reply:
Running the file wont be an issue, but if we expect this file to grow at all it'll become more difficult to work with. Our specs are in general too long, so I thought I'd ask the question.

Reply:
Ok, it might be wise to do this at this point then with DLS-related changes potentially causing files to increase.

Reply:
I can always split file, but in this case it provides inconsistency for cucumber scenarios in general. If Alert feature will grow then we can think about splitting scenarios. In my opinion 130 lines feature file is ok. I believe we have coding standards for java script so for cucumber feature files we can agree coding standards too

Reply:
I isolate Given steps to Background to reduce lines of code